plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/MongoServer.java

```diff
@@ -44,6 +45,11 @@ public HostAndPort getAddress()
         return HostAndPort.fromParts(dockerContainer.getContainerIpAddress(), dockerContainer.getMappedPort(MONGO_PORT));
     }
 
+    public ConnectionString getConnectionString()
```

Comment:
Please use this method from `TestMongoPlugin` and then remove `MongoServer.getAddress` method.